cms/models/pagemodel.py

```diff
@@ -791,7 +791,7 @@ def get_published_languages(self):
 
     def get_cached_ancestors(self):
         if not hasattr(self, "ancestors_ascending"):
-            self.ancestors_ascending = list(self.get_ancestors())
+            self.ancestors_ascending = list(reversed(self.get_ancestors()))
```

Comment:
won't this make ``get_cached_ancestors`` inconsistent with ``get_ancestors``? Looking at their names I would assume they both return the same thing.